credsweeper/ml_model/ml_validator.py

```diff
@@ -86,35 +91,35 @@ def encode(cls, line, char_to_index) -> np.ndarray:
         return one_hot[0]
 
     @classmethod
-    def extract_common_features(cls, candidates: List[Candidate]) -> np.ndarray:
+    def extract_common_features(cls, candidates: List[Candidate]) -> 'np.ndarray':
         """Extract features that are guaranteed to be the same for all candidates on the same line with same value."""
-        features = np.array([], dtype=float)
+        np_array = np.array([], dtype=float)
         # Extract features from credential candidate
         default_candidate = candidates[0]
         for feature in cls.common_feature_list:
             new_feature = feature([default_candidate])[0]
             if not isinstance(new_feature, np.ndarray):
                 new_feature = np.array([new_feature])
-            features = np.append(features, new_feature)
-        return features
+            np_array = np.append(np_array, new_feature)
+        return np_array
 
     @classmethod
-    def extract_unique_features(cls, candidates: List[Candidate]) -> np.ndarray:
+    def extract_unique_features(cls, candidates: List[Candidate]) -> 'np.ndarray':
         """Extract features that can by different between candidates. Join them with or operator."""
-        features = np.array([], dtype=bool)
+        np_array = np.array([], dtype=bool)
```

Comment:
```suggestion
        feature_array = np.array([], dtype=bool)
```

`np_array` is a bit too general statement :)
Other than that LGTM, sorry for bothering